engine/tests/test_lua10_lust_eval.lua

```diff
@@ -0,0 +1,86 @@
+local cloe = require("cloe")
+local project = cloe.require("project")
+local lust = require 'lust'
+local it, expect = lust.it, lust.expect
+
+project.init_report()
+
+cloe.apply_stack {
+    version = "4",
+    include = {
+        "config_nop_infinite.json",
+    },
+    server = {
+        listen = false,
+        listen_port = 23456,
+    },
+}
+
+--cloe.init_report()
+
+-- All the conditions we want to fail on:
+cloe.schedule_these {
+    run = "fail",
+    { on = "virtue/failure" },
+    { on = "default_speed/kmph=>0.0" },
+    { on = "time=5" },
+}
+
+-- All the things we want to do on start:
+cloe.schedule_these {
+    on = "start",
+    { run = "log=info: Running nop/basic smoketest." },
+    { run = "realtime_factor=-1" },
+}
+
+cloe.schedule { on = "time=60", run = "succeed" }
+
+cloe.schedule {
+    on = "loop",
+    pin = true,
+    run = function(sync)
+        cloe.log("info", "Current time is %s", sync:time())
+    end
+}
+
+-- schedule_test with improved logging using LUST - https://github.com/bjornbytes/lust
+cloe.schedule_test {
+    -- Note that this is the same ID as used in BATS.
+    id = "e03fc31f-586b-4e57-80fa-ff2cba5ff9de",
+    on = "start",
+    info={hello= "test", rqm="big number"},
```

Comment:
If you don't have a lua formatter set up, now's a good time to do that. Then apply here.